qemu/tests/multi_nics_verify.py

```diff
@@ -81,7 +81,7 @@ def check_nics_num(expect_c, session):
                 test.error(err_msg % (eth_config_path, o))
 
         # Reboot and check the configurations.
-        session = vm.reboot(session)
```

Comment:
You can reuse the login_timeout in L55.

Reply:
> You can reuse the login_timeout in L55.

Thanks for you suggestion, updated. Please help review it again.